client/packages/common/src/ui/components/inputs/DatePickers/DateTimePickerInput/DateTimePickerInput.tsx

```diff
@@ -22,6 +22,7 @@ export const DateTimePickerInput: FC<
     onChange: (value: Date | null) => void;
     onError?: (validationError: string, date?: Date | null) => void;
     textFieldProps?: TextFieldProps;
+    isDate?: boolean;
```

Comment:
:nit a naming confusion - because the DateTimePickerInput always shows date, this would be more helpful if it was `isOnlyDate` 

if it were me, I would have a prop for `displayTime` or `showTime` (which is funnier, because it sounds like 'showtime! :jazz hands:')

Reply:
Ooh brilliant! Using showTime :D